MWSE/LuaObject.cpp

```diff
@@ -23,6 +23,8 @@ namespace mwse::lua {
 			return std::make_unique<ObjectCreator<TES3::Enchantment>>();
 		case TES3::ObjectType::Weapon:
 			return std::make_unique<ObjectCreator<TES3::Weapon>>();
+		case TES3::ObjectType::Book:
+			return std::make_unique<ObjectCreator<TES3::Book>>();
 		default:
```

Comment:
This isn't necessary, but what about keeping the switch case sorted alphabetically, and adding this below `TES3::ObjectType::Ammo`?

Reply:
Yeah I'll move it there. Enchantment should probably also be moved.